test/libsolidity/syntaxTests/literalSuffixes/parameters/mantissa_out_of_range.sol

```diff
@@ -29,24 +28,27 @@ contract C {
         65536_000 u16;
         57896044618658097711785492504343953926634992332820282019728792003956564819968_000 s256; // 2**255 * 10**-76
         115792089237316195423570985008687907853269984665640564039457584007913129639936_000 u256; // 2**256 * 10**-77
+
+        128e-256 mi8eu8;
     }
 }
 // ----
-// TypeError 8838: (559-565): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (575-581): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (591-600): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (610-619): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (629-711): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 5503: (731-814): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function.
-// TypeError 8838: (835-842): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (852-859): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (869-879): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (889-899): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (909-992): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 5503: (1022-1106): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function.
-// TypeError 8838: (1137-1147): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (1157-1167): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (1177-1190): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 8838: (1200-1213): The type of the literal cannot be converted to the parameters of the suffix function.
-// TypeError 5503: (1223-1309): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function.
-// TypeError 5503: (1339-1426): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function.
+// TypeError 8838: (475-481): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of the suffix function. The mantissa is out of range of type int8.
```

Comment:
The number in this case is an integer. We should drop "fractional" from the message:
```suggestion
// TypeError 8838: (475-481): This number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of the suffix function. The mantissa is out of range of type int8.
```

Same for other, similar messages.